### DISCUSSION TITLE
change default Settings for remote menu items

### DISCUSSION THREAD

**Discussion by nikkdevic** [rustdesk/rustdesk#1978] [General]
Currently the default options are whatever is set in the software. This settings menu can let users change default behaviour.
Some Suggested Options -
1. Default View - Shrink, Original, Stretch
2. Default Allow Copy and Paste - Yes, No

After the menu is implemented, the Default Options from this menu will only apply on first connection to new remote system and then changes can be remembered for each system as they are remembered now.

**Comment by rustdesk** [comment]
Please put your "should" and "like" in discussions rather than issues

**Reply by nikkdevic** [reply]
ok 👍 
But this is a feature request for a Settings Menu

**Reply by rustdesk** [reply]
your "should" / "like" feature is not normal feature

**Reply by rustdesk** [reply]
how about another guy has anther "should" and "like" about this topic?

**Reply by nikkdevic** [reply]
Currently the default options are whatever is set in the software. This settings menu can let users change default behaviour. So another person can customize from this menu as they "like".

**Reply by rustdesk** [reply]
agree, i got the same idea yesterday.

**Reply by nikkdevic** [reply]
Great 👍

**Reply by rustdesk** [reply]
change default per your latest selection or in settings, which one is better? i do not know, both have their advantage.

**Reply by nikkdevic** [reply]
> change default per your latest selection or in settings, which one is better? i do not know, both have their advantage.

You can add that in the settings too

**Reply by rustdesk** [reply]
we are too busy, you can not see it in the near future, but PR welcome.

**Reply by nikkdevic** [reply]
I think settings is better because -
1. The default apply to new connections only.
2. Settings changed in one connection shouldn't affect other connection. Remember settings for each connection separately as it currently does.

**Reply by nikkdevic** [reply]
> we are too busy, you can not see it in the near future, but PR welcome.

Thats OK 👍. Unfortunately i only do some basic coding. If we move this to issue someone might PR

**Reply by rustdesk** [reply]
everyone is expecting the others

**Reply by nikkdevic** [reply]
I understand, but i dont know how.

**Reply by nikkdevic** [reply]
![AnydeskSettings](https://user-images.githubusercontent.com/97282126/200154102-99e03d1e-b90e-434d-a224-9a99fe9afc2f.png)

**Reply by rustdesk** [reply]
Thanks, a good reference

**Reply by rustdesk** [reply]
it is now in our mikestone https://github.com/rustdesk/rustdesk/discussions/918

**Reply by nikkdevic** [reply]
👍
